app/javascript/models/User.tsx

```diff
@@ -0,0 +1,35 @@
+import { observable, computed } from 'mobx';
+import { User as UserProps } from '../api/user';
```

Comment:
well, this rather a convention question, but I would rather move the UserProps to the user model file and import the props in the user API than the opposite.

Reply:
hmm... in my understanding the `api/user` is glued to the api - when the api changes, the user at this place has to change too (with it's api-based snake-case notation). At `models/User` we live in the JS-World (and have camelCase notations...) so i prefere to split those conecpts...